ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/host1/aaa-accounting.yml

```diff
@@ -16,8 +16,15 @@ aaa_accounting:
       group: TACACS
   dot1x:
     default:
-      type: start-stop
-      group: RADIUS
+      type: stop-only
```

Comment:
It seems all the test covered with respect to coverage but please do confirm by coverage report